src/users/users.service.ts

```diff
@@ -7,6 +7,7 @@ import { hash } from 'bcrypt';
 import { UserEntity } from '../database/entity/user.entity';
 import { DatabaseError } from '../common/database-error';
 import { CreateUserDto, UpdateUserDto } from './dto/user.dto';
+import { ConfigService } from '@nestjs/config';
```

Comment:
A config service-t felhasználod a fájlban? mert látszólag nem, viszont akkor felesleges az import is